raiden/channel/netting_channel.py

```diff
@@ -646,7 +640,7 @@ def create_directtransfer(self, amount, identifier):
         This message needs to be signed and registered with the channel before
         sent.
         """
-        if not self.isopen:
+        if not self.can_transfer:
             raise ValueError('The channel is closed')
```

Comment:
The error message implies that `close` was called, but maybe it's just lacking funds, could you improve it?